packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -50,64 +48,59 @@ export class OrgList implements vscode.Disposable {
+    const orgAuthorizations = await AuthInfo.listAllAuthorizations().catch(
       err => null
     );
 
-    if (authFilesArray === null || authFilesArray.length === 0) {
+    if (orgAuthorizations === null || orgAuthorizations.length === 0) {
       return null;
     }
-    const authInfoObjects: FileInfo[] = [];
-    for (const authFile of authFilesArray) {
-      try {
-        const filePath = path.join(
-          await ConfigFile.resolveRootFolder(true),
-          '.sfdx',
-          authFile.username + '.json'
-        );
-        const fileData = readFileSync(filePath, 'utf8');
-        authInfoObjects.push(JSON.parse(fileData));
-      } catch (e) {
-        console.log(e);
-      }
-    }
-    return authInfoObjects;
+
+    return orgAuthorizations;
   }
 
-  public async filterAuthInfo(authInfoObjects: FileInfo[]) {
-    authInfoObjects = authInfoObjects.filter(
-      fileData => !fileData.scratchAdminUsername
-    );
+  public async getAuthFieldsFor(username: string): Promise<AuthFields> {
+    const authInfo: AuthInfo = await AuthInfo.create({
+      username
+    });
+    return authInfo.getFields();
+  }
 
+  public async filterAuthInfo(orgAuthorizations: OrgAuthorization[]) {
     const defaultDevHubUsernameorAlias = await this.getDefaultDevHubUsernameorAlias();
+    let defaultDevHubUsername: string | undefined;
     if (defaultDevHubUsernameorAlias) {
-      const defaultDevHubUsername = await OrgAuthInfo.getUsername(
+      defaultDevHubUsername = await OrgAuthInfo.getUsername(
         defaultDevHubUsernameorAlias
       );
-
-      authInfoObjects = authInfoObjects.filter(fileData => {
-        const isNullOrUndefined = !fileData.devHubUsername;
-        return (
-          isNullOrUndefined ||
-          (!isNullOrUndefined &&
-            fileData.devHubUsername === defaultDevHubUsername)
-        );
-      });
     }
 
-    const info = await StateAggregator.create();
     const authList = [];
     const today = new Date();
-    for (const authInfo of authInfoObjects) {
-      const aliases = info.aliases.getAll(authInfo.username);
-      const isExpired = authInfo.expirationDate
-        ? today >= new Date(authInfo.expirationDate)
+    for (const orgAuth of orgAuthorizations) {
+      const authFields: AuthFields = await this.getAuthFieldsFor(
+        orgAuth.username
+      );
+      if (authFields?.scratchAdminUsername) {
+        // non-Admin scratch org users
+        continue;
+      }
+      if (
+        authFields?.devHubUsername &&
+        authFields.devHubUsername !== defaultDevHubUsername
+      ) {
+        // scratch orgs parented by other (non-default) devHub orgs
+        continue;
+      }
+      const isExpired = authFields?.expirationDate
+        ? today >= new Date(authFields.expirationDate)
         : false;
+
       let authListItem =
-        aliases.length > 0
-          ? `${aliases} - ${authInfo.username}`
-          : authInfo.username;
```

Comment:
authInfo wasn't providing all (or any) of the Aliases correctly

Reply:
AuthFields.alias? wasn't returning the expected values, so went back to using OrgAuthorization.aliases.